test/integration/component/test_vpc_network.py

```diff
@@ -2748,3 +2751,86 @@ def test_stop_start_vpc_router(self):
         if (exceptionOccured or (not isRouterInDesiredState)):
             self.fail(exceptionMessage)
         return
+
+    @attr(tags=["advanced", "intervlan"], required_hardware="true")
+    def test_restart_network_with_cleanup(self):
+        """ Test restart network with cleanup
+
+        #1.Acquire pubilc ip
+        #2.Configure PF rule on acquired ip
+        #3.Restart network wich leanup
+        #4.Verify that IP won't be deleted from vpc VR
+        """
+        self.public_ip_1 = PublicIPAddress.create(
+            self.apiclient,
+            accountid=self.account.name,
+            zoneid=self.zone.id,
+            domainid=self.account.domainid,
+            networkid=self.network_1.id,
+            vpcid=self.vpc.id
+        )
+        NATRule.create(
+            self.apiclient,
+            self.vm_1,
+            self.services["natrule"],
+            ipaddressid=self.public_ip_1.ipaddress.id,
+            networkid=self.network_1.id
+        )
```

Comment:
@sanju1010  Can we add a validation to make sure that secondary IP is indeed present on the VR even before restarting the network , so that the test will not miss out on the issue where secondary IP was not present on the VR interface even before restart for some reason ?